core/lib/mcxtx_api.h

```diff
@@ -128,6 +128,7 @@
         uint apsr; /**< The application program status registers in AArch32. */
         uint cpsr; /**< The current program status registers in AArch32. */
     }; /**< The anonymous union of alternative names for apsr/cpsr register. */
+    byte padding[4]; /**< The padding to get simd field 8-byte aligned. */
```

Comment:
Changing the layout of a public struct is a major compatibility break requiring bumping the major version, first done by immediately jumping to 11.99 per https://dynamorio.org/page_new_release.html#autotoc_md231.  So this PR needs to jump to 11.99 and needs to list this change in the compatibility break changelist in release.dox.  (If there's an alternative solution that doesn't require breaking compatibility that could be weighed against the breakage.)

Reply:
You mean 11.90?

Reply:
Oh, whoops, yes, 11.90.